src/restLevelClient/generateRestLevel.ts

```diff
@@ -43,6 +48,13 @@ export async function generateRestLevelClient() {
   generateLicenseFile(project);
   generateTsConfig(project);
   generateApiExtractorConfig(project);
+
+  generateKarmaConfigFile(project)
+  generateEnvFile(project);
+  generateEnvBrowserFile(project);
+  generateRecordedClientFile(project);
+  generateSampleTestFile(project);
+
```

Comment:
add a if generateTest is set then generate those logic here ?

Reply:
there is if in functions